message_definitions/v1.0/common.xml

```diff
@@ -1471,6 +1474,16 @@
         <param index="6">Empty</param>
         <param index="7">Empty</param>
       </entry>
+      <entry value="198" name="MAV_CMD_DO_SET_ROI_SYSID">
+        <description>Follow vehicle by system ID.</description>
```

Comment:
Does this mean follow as in "Follow me" or "Camera ROI is vehicle with specified SysID"?

Reply:
Swiping that better description ("Camera ROI is vehicle with specified SysID"), thanks.